Make image_downloader docs run on CI

xnutsive opened the pull request:
@sgugger, not sure if I can push to an existing & merged PR, created a new one. 

Changes: 
- I fixed the `docs_src/`

stas00 commented:
When you click on the failure details: https://github.com/fastai/fastai/pull/1427/checks?check_run_id=46187666

Do you get the option to Re-run all?
![screenshot_6](https://user-images.githubusercontent.com/10676103/50712208-80aa7580-1025-11e9-9332-54c6a2b9369b.png)

It shouldn't be there for anybody but those with commit rights, but perhaps the PR submitter has that option too? Let us know if you don't and we will re-run it.

sgugger commented:
I have an output error in the cell after `learn.fit_one_cycle(3)` (keyboard interrupt, not the end of the word, but if you could commit a clean file, it'd be best).
Also, the cell with show_doc doesn't seem to be hidden (it may be nbdiff not catching it though).

xnutsive commented:
Sorry for the delay, yep, I’ll clean things up and commit on Tuesday.

On Fri, Jan 4 2019 at 1:43 PM, < notifications@github.com > wrote:

> 
> 
> 
> I have an output error in the cell after learn.fit_one_cycle(3) (keyboard
> interrupt, not the end of the word, but if you could commit a clean file,
> it'd be best).
> Also, the cell with show_doc doesn't seem to be hidden (it may be nbdiff
> not catching it though).
> 
> 
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub (
> https://github.com/fastai/fastai/pull/1427#issuecomment-451577274 ) , or mute
> the thread (
> https://github.com/notifications/unsubscribe-auth/AACqcSFZuX6jwg49lWN0YyeXOV7aPq8iks5u_8sTgaJpZM4ZpESX
> ).
> 
> 
>

xnutsive commented:
Alright I think I fixed it, fixed the keyboard interrupt and moved the undocumented functions in the hidden section.

Most of them are for image cleaner. I’ll open an issue for Francisco and if his busy, I’ll write some docs in the weekend. Would that be ok?

On Mon, Jan 7 2019 at 10:01 PM, Nate Gadzhibalaev < nate@respawn.io > wrote:

> 
> 
> Sorry for the delay, yep, I’ll clean things up and commit on Tuesday.
> 
> 
> 
> On Fri, Jan 4 2019 at 1:43 PM, < notifications@github.com > wrote:
> 
>> 
>> 
>> I have an output error in the cell after learn.fit_one_cycle(3) (keyboard
>> interrupt, not the end of the word, but if you could commit a clean file,
>> it'd be best).
>> Also, the cell with show_doc doesn't seem to be hidden (it may be nbdiff
>> not catching it though).
>> 
>> 
>> 
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub (
>> https://github.com/fastai/fastai/pull/1427#issuecomment-451577274 ) , or mute
>> the thread (
>> https://github.com/notifications/unsubscribe-auth/AACqcSFZuX6jwg49lWN0YyeXOV7aPq8iks5u_8sTgaJpZM4ZpESX
>> ).
>> 
>> 
>> 
> 
>

xnutsive commented:
@sgugger thank you for patiently reviewing and nudging me on this ;-)
